crates/fluvio-cli/src/connector/mod.rs

```diff
@@ -165,23 +165,35 @@ impl From<ConnectorConfig> for ManagedConnectorSpec {
         if let Some(producer) = config.producer {
             if let Some(linger) = producer.linger {
                 let linger = humantime::format_duration(linger).to_string();
-                parameters.insert("producer-linger".to_string(), linger);
+                parameters.insert(
+                    "producer-linger".to_string(),
+                    ManageConnectorParameterValue::from(linger),
```

Comment:
nit: Maybe linger.into() is more cleaner

Reply:
Yeah it is.